src/modules/Messages.js

```diff
@@ -0,0 +1,17 @@
+import { DataTypes } from 'sequelize';
+import { client } from '../utils/db.js';
+import { Room } from './Room.js';
+
+export const Messages = client.define('messages', {
+  text: {
+    type: DataTypes.STRING,
+    allowNull: false,
+  },
+  author: {
+    type: DataTypes.STRING,
+    allowNull: false,
+  },
+});
+
+Messages.belongsTo(Room);
+Room.hasOne(Messages);
```

Comment:
The relationship `Room.hasOne(Messages)` might not be appropriate if a room is expected to have multiple messages. Consider using `Room.hasMany(Messages)` to reflect a one-to-many relationship.